src/angular-app/bellows/apps/userprofile/user-profile-app.component.ts

```diff
@@ -203,6 +217,58 @@ export class UserProfileAppController implements angular.IController {
     });
   }
 
+  checkIfUserOwnsAProject(){
+    this.ownsAProject = false;
+    if(this.projects.length > 0){
+      this.projects.forEach(project => {
+        if(project.ownerId === this.user.id){
+          this.ownsAProject = true;
+        }
+      });
+    }
+  }
+
+  queryProjectsForUser() {
+    this.finishedLoadingOwnedProjects = false;
+    this.projects = [];
+    this.ownedProjects = [];
+    this.projectService.list().then((projects: Project[]) => {
+      this.projects = projects || [];
+      this.checkIfUserOwnsAProject();
+      if(this.ownsAProject){
+        this.findOwnedProjects();
+      }
```

Comment:
I think you can simplify this a fair bit:
```suggestion
      this.ownedProjects = projects.filter(proj => proj.ownerId === this.user.id);
      this.ownsAProject  = this.ownedProjects.length > 0;
```

Reply:
Nice! 🎉 committing this from my IDE and removing code for the functions, too; they won't be needed